packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -165,12 +180,14 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         // width={this.props.style.chartWidth}
         return <React.Fragment>
             {this.state.outputStatus === ResponseStatus.COMPLETED ?
-                <div id='xy-main' tabIndex={0}
+                <div id={this.getUniqueDivId()} tabIndex={0}
```

Comment:
I am not sure if this change is required anymore now that we have the chartRef. Could you check if it works without it?

Reply:
i just tried now and it works.